playbooks/generic-clean_airship.yml

```diff
@@ -15,13 +15,13 @@
 # under the License.
 #
 
-- hosts: airship-deployer
+- hosts: soc-deployer
   gather_facts: no
   tasks:
     - name: Running cleanup script
       script: "{{ playbook_dir }}/../script_library/clean-airship.sh {{ clean_action | default('') }}"
 
-- hosts: airship-workers
+- hosts: caasp-workers
```

Comment:
I think we should make this file not only a airship thing, but a "clean my caasp setup of all those pesky helm charts" ;) 

For a different commit though.